### ISSUE TITLE
Create an address book record

### ISSUE THREAD

**Issue by kborovik** [logictimes/privatekey.space#3]
## As an (actor) 
an end user
## I want (action)
to create an address book record
## so that (reason)
I can add a contract participant contact with the following fields:
- Name
- Email
- Extended PUB key

**Comment by alienatix** [comment]
@kborovik in all stories except for "log in" I'd recommend that we use role "authenticated user" or something like that because this functionality should not be available to any users who are not authenticated using Trezor.

**Comment by alienatix** [comment]
@kborovik additional requirements for a Contact entry (though you may decide to skip it for MVP):
* all fields are mandatory
* each record should be unique
* an appropriate error message should be shown when a validation fails

**Comment by kborovik** [comment]
@alienatix, agreed. I will correct description of the user to "authenticated user"

**Comment by kborovik** [comment]
@alienatix, I will add fields requirements.
